analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -41,13 +42,21 @@ import com.here.ort.utils.realFile
 import java.io.File
 import java.time.Instant
 
+import kotlinx.coroutines.ObsoleteCoroutinesApi
+import kotlinx.coroutines.async
+import kotlinx.coroutines.coroutineScope
+import kotlinx.coroutines.newFixedThreadPoolContext
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.withContext
+
 const val TOOL_NAME = "analyzer"
 const val HTTP_CACHE_PATH = "$TOOL_NAME/cache/http"
 
 /**
  * The class to run the analysis. The signatures of public functions in this class define the library API.
  */
 class Analyzer(private val config: AnalyzerConfiguration) {
+    @ObsoleteCoroutinesApi
```

Comment:
Can we please repeat the comment from `scanPackages` about the obsolete API here?

Reply:
Actually, is the annotation required here, or is it sufficient to have it below?

Reply:
I'm using a different way to create the dispatcher now, not using the obsolete API anymore. But the annotation was actually required in all places I used it.